jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -2178,17 +2181,8 @@ public AsyncContext startAsync(ServletRequest servletRequest, ServletResponse se
         HttpChannelState state = getHttpChannelState();
         if (_async == null)
             _async = new AsyncContextState(state);
-        AsyncContextEvent event = new AsyncContextEvent(_context, _async, state, this, servletRequest, servletResponse);
+        AsyncContextEvent event = new AsyncContextEvent(_context, _async, state, this, servletRequest, servletResponse, true);
```

Comment:
I wonder if you shouldn't pass the `providedUri` here instead of a boolean.

Reply:
ack